db/migrate/20170511120047_remove_container_group_failed_sync_event.rb

```diff
@@ -0,0 +1,14 @@
+class RemoveContainerGroupFailedSyncEvent < ActiveRecord::Migration[5.0]
+  class Relationship < ActiveRecord::Base; end
+
+  class MiqEventDefinition < ActiveRecord::Base; end
+
+  def up
+    say_with_time("Remove event definition containergroup_failedsync") do
+      MiqEventDefinition.where(:name => 'containergroup_failedsync').each do |eventdef|
+        Relationship.where(:resource_type => 'MiqEventDefinition', :resource_id => eventdef.id).delete_all
```

Comment:
I think this means there is a miq event definition set and this event definition belongs to a set ?